pkg/kudoctl/cmd/install/install.go

```diff
@@ -4,10 +4,13 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/google/martian/log"
```

Comment:
Not sure if we should introduce this here and stay with printing it out for `0.2.0`